packages/cf-builder-card/src/cardReducer.js

```diff
@@ -1,10 +1,12 @@
-const ActionTypes = require('./CardActionTypes');
+// @flow
 
-const initialState = {};
+import type {CardAction, CardState} from './CardTypes';
```

Comment:
I think the takeaway is, when everything is bundled, the modules will be ordered and eval'ed statically except things loaded by `import()`, in which Webpack will wrap into an "installable" static order anyway. We don't effect any side-effect in module exports that can't be replicated statically, and we certainly don't have any circular imports. There doesn't seem to be any proposed changes that will affect us.